src/routes/members/gigs/[gig_id([0-9]+)]/edit.svelte

```diff
@@ -0,0 +1,1133 @@
+        {#each clients as contact (contact.id)}
+          <div class="gig-contact" transition:fade|local>
+            <span data-test="contact-name">{contactDisplayName(contact.contact)}</span>
+            <div class="button-group">
+              <button
+                on:click="{editContact(contact.id, contactTypes.CLIENT)}"
+                data-test="gig-edit-{id}-clients-{contact.id}-edit"
+                disabled="{cancelled}"
+              >Edit</button>
+              <button
+                on:click="{removeClient(contact.id)}"
+                data-test="gig-edit-{id}-clients-{contact.id}-remove"
+                disabled="{cancelled}"
+              >Remove</button>
+            </div>
+          </div>
+        {/each}
+      </div>
+      <!-- svelte-ignore a11y-label-has-associated-control -->
+      <label data-test="gig-edit-{id}-client-select">Add client
+        <Select bind:value="{selectedClient}" bind:select="{clientListElement}" disabled="{cancelled}">
+          <option selected="selected" disabled value="{undefined}">--- SELECT A CLIENT ---</option>
+          {#each potentialClients as contact}
+            <option value="{contact.id}">{contactDisplayName(contact)}</option>
+          {/each}
+        </Select></label>
+      <SearchBox
+        placeholder="Search clients"
+        fuse="{clientFuse}"
+        toId="{(client) => client.id}"
+        toDisplayName="{contactDisplayName}"
+        on:select="{selectClientSearch}"
+        data-test="gig-edit-{id}-client"
+        disabled="{cancelled}"
+      />
+      <div class="button-group">
+        <button on:click="{selectClient}" data-test="gig-edit-{id}-client-select-confirm" disabled="{cancelled}">Select
+          client</button>
+        <button on:click="{newClient}" data-test="gig-edit-{id}-client-new" disabled="{cancelled}">Create new client</button>
+      </div>
+    </form>
+    {#if typeCode !== 'kit'}
+      <h4>Callers</h4>
+      <form on:submit|preventDefault class="theme-{$themeName} contacts" data-test="caller-form">
+        <div data-test="gig-edit-{id}-caller-list">
+          {#each callers as contact (contact.id)}
+            <div class="gig-contact" transition:fade|local>
+              <span data-test="contact-name">{contactDisplayName(contact.contact)}</span>
+              <div class="button-group">
+                <button
+                  on:click="{editContact(contact.id, contactTypes.CALLER)}"
+                  data-test="gig-edit-{id}-callers-{contact.id}-edit"
+                  disabled="{cancelled}"
+                >Edit</button>
+                <button
+                  on:click="{removeCaller(contact.id)}"
+                  data-test="gig-edit-{id}-callers-{contact.id}-remove"
+                  disabled="{cancelled}"
+                >Remove</button>
+              </div>
+            </div>
+          {/each}
+        </div>
+        <!-- svelte-ignore a11y-label-has-associated-control -->
```

Comment:
what is this for?

Reply:
To prevent a false positive warning (it can't detect form controls in components)